docs/cli.md

```diff
@@ -193,7 +197,7 @@ The following options are supported:
 The user can edit the parts of the generated OSCAL model by modifying the sample content in those directories.
 
 The initial level of decomposition of each type of model varies according to the model type.
-This default or reference decomposition behaviour can be changed by modifying the rules in a `.trestle/config file`. These rules can be written as a sequence of `trestle split` commands.
+This default or reference decomposition behaviour can be changed by modifying the rules in `.trestle/config.ini` file. These rules can be written as a sequence of `trestle split` commands.
```

Comment:
I think we want to remove all references to config.ini

Reply:
Removed this para completely as during create only one .json file is created and not a split model.